contrib/ipp/CMakeLists.txt

```diff
@@ -30,11 +30,18 @@ set( SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src" )
 # Optimized for processors with Intel SSE4.2
 
 if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-  set( defs "-D_Y8 -D_ARCH_EM64T" )
+  append( "-D_Y8" CMAKE_C_FLAGS )
```

Comment:
don't use append, please. The following code will be better I believe:
```
if(CMAKE_SIZEOF_VOID_P EQUAL 8)
    set( ipp_defs "-D_Y8 -D_ARCH_EM64T" )
else()
    set( ipp_defs "-D_P8 -D_ARCH_IA32" )
endif()
list(APPEND CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ipp_defs}")
```
mind change of `defs` to `ipp_defs` since msdk build system reacts on `defs` by itself.

Reply:
done